src/main/java/de/danielbechler/diff/access/PropertyAwareAccessor.java

```diff
@@ -29,4 +29,8 @@
 	Set<Annotation> getReadMethodAnnotations();
 
 	<T extends Annotation> T getReadMethodAnnotation(Class<T> annotationClass);
+
+   <T extends Annotation> T getAnnotation(Class<T> annotationClass);
+
+   int getFieldModifiers();
```

Comment:
I followed your idea about the custom Introspector. All tests passed and I don't need to change your code anymore. I took also the ArrayDiffer + Factory into my code base so that I can work now with the standard version 0.91.1. Thanks for your help!